.readthedocs.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
```

Comment:
2023/9/25より、ReadTheDocsの設定画面が一部廃止され、設定ファイル必須となります。

by https://blog.readthedocs.com/migrate-configuration-v2/'

> Migrate your project to .readthedocs.yaml configuration file v2[¶](https://blog.readthedocs.com/migrate-configuration-v2/#migrate-your-project-to-readthedocs-yaml-configuration-file-v2)
> We are announcing a new requirement for all builds to use our configuration file version 2. This announcement deprecates builds without a configuration file, as well as version 1 of our configuration file.
> 
> Read the Docs will start requiring a .readthedocs.yaml configuration file for all projects in order to build documentation successfully. We will stop supporting builds without explicit configuration, because this creates implicit dependencies that users aren’t aware of. We plan to start failing builds not using configuration file version 2 on September 25, 2023.


refs: https://docs.readthedocs.io/en/stable/config-file/v2.html#python-install